README.md

```diff
@@ -73,10 +73,10 @@ Flags:
       --max-extraction-size=1073741824     Maximum extraction size that allowed is (in bytes). (disable check: -1)
       --max-extraction-time=60             Maximum time that an extraction should take (in seconds). (disable check: -1)
       --max-input-size=1073741824          Maximum input size that allowed is (in bytes). (disable check: -1)
+  -P, --no-preserve-file-attributes        Do not preserve file attributes (mode, modtime, access time).
```

Comment:
I think this is a good change, make the default to preserve attributes but keep the zero value of bool as the default.

Reply:
Over night, I thought it might be an idea to rename the option to "drop file attributes" for more clearance.

Reply:
Renamed the flag in https://github.com/hashicorp/go-extract/pull/106/commits/5ff62798be4f469be5c42b1c77c3a23f40ca5a5d.